src/main/kotlin/Main.kt

```diff
@@ -1,11 +1,21 @@
-import Calculator.Calculator
+import lotto.view.InputView
+import lotto.domain.LottoGame
+import lotto.view.ResultView
 
 fun main() {
     try {
-        println("계산하기 위한 문자열을 입력해주세요.")
-        val calculator = Calculator(readLine()!!)
-        val result = calculator.execute()
-        println("총합은 $result 입니다.")
+        val gameMoney = InputView.getGameMoney()
+        val lotto = LottoGame(gameMoney)
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-lotto/pull/26#discussion_r465395531

```suggestion
        val lottoGame = LottoGame(gameMoney)
```